src/main/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxy.scala

```diff
@@ -155,7 +155,8 @@ class InMemoryWorldStateProxy private(
     copyWith(accountsStateTrie = accountsStateTrie.put(address.bytes, account))
 
   override def deleteAccount(address: Address): InMemoryWorldStateProxy =
-    copyWith(accountsStateTrie = accountsStateTrie.remove(address.bytes))
+    copyWith(accountsStateTrie = accountsStateTrie.remove(address.bytes),
+      contractStorages = contractStorages - address)
```

Comment:
It tried to persist contract storage (of contract that no longer exists) in `persist`. Here's the exact line: https://github.com/input-output-hk/etc-client/blob/86bf3ad6fd1dcbe7f81d41a5025e22f2d63b2a5a/src/main/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxy.scala#L67